packages/@react-aria/tooltip/src/useTooltipTrigger.ts

```diff
@@ -129,8 +129,6 @@ export function useTooltipTrigger(props: TooltipTriggerProps, state: TooltipTrig
     onHoverEnd
   });
 
-  let {pressProps} = usePress({onPressStart});
```

Comment:
given that the usePress logic for onPressStart isn't particularly difficult (compared to the other press events), we could consider just implementing this tooltip close as
onPointerDown/onMouseDown/onKeyDown/etc directly?

Reply:
Definitely tempting since we wouldn't break the experience for users who were relying on usePress being called for them in useToolTipTrigger. Bit wary of doing that though since we would be duplicating the usePress logic (we would need to do the stop propagation stuff at the very least) and thus drift could happen. I'm still leaning towards passing `onPressStart` down, feels like the "correct" thing to do since the trigger element should be pressable/focusable and thus should be calling `usePress`/`useFocusable` itself or via our `useButton`/`useLink` hooks

Reply:
if we do that, we'd need to release it as a major

would we actually need to do the stop propagation? why? tooltips are a little strange, clicking the trigger should close the tooltip, but it should also do whatever action the button does when clicked. The usePress on the button would then do any of the stops or prevent defaults for us. Or have I not quite understood the order of events here?

Reply:
> if we do that, we'd need to release it as a major

argh, that's annoying

> would we actually need to do the stop propagation?

If were were to replace the usePress with onPointerDown/onMouseDown/onKeyDown/etc then we should probably stop propagation to keep the behavior consistent since users may have been inadvertently relying on that quirk of usePress. If the user uses `useTooltipTrigger` with a trigger element that has `usePress`/`useButton`/etc then we wouldn't have to stop propagation in onPointerDown/onMouseDown/onKeyDown/etc but my concern is if the user was just passing the `triggerProps` directly onto a [button like it shows in the docs](https://react-spectrum.adobe.com/react-aria/useTooltipTrigger.html#example).

Reply:
I see your point. It's annoying that it's been that way so long because I'd argue it was a mistake and wrong since we are only using press start. But it's hard to argue that when it's been so for ~3 years. We should gather a few more opinions.